### ISSUE TITLE
Ivy compilation error (Angular 9 rc.11)

### ISSUE THREAD

**Issue by hevans90** [okta/okta-oidc-js#655]
## I'm submitting this issue for the package(s):

- [ ] jwt-verifier
- [x] okta-angular
- [ ] oidc-middleware
- [ ] okta-react
- [ ] okta-react-native
- [ ] okta-vue

## I'm submitting a:

- [x] Bug report  
- [ ] Feature request
- [ ] Other (Describe below)

## Current behavior
After an upgrade from Angular 9 rc.10 to rc.11, the following error occurs upon compilation of an app consuming the `OktaAuthModule`:

![image](https://user-images.githubusercontent.com/12142999/73169619-6346be80-40f4-11ea-8906-b4e909e12aa7.png)

## Expected behavior
It compiles succesfully.

## Minimal reproduction of the problem with instructions

- Create a new angular project with Angular 9 rc.11
- `yarn add @okta/okta-angular@1.3.1`
- Import the `OktaAuthModule` in to the `AppModule`
- `ng serve`

## Environment

- Package Version: `@okta/okta-angular@1.3.1`
- Browser: n/a
- OS: OSX Catalina
- Node version (`node -v`): 13.6.0
- Other:

**Comment by hevans90** [comment]
Some issues from other libraries that seem similar:

https://github.com/flauc/angular2-notifications/issues/355
https://github.com/bazelbuild/rules_nodejs/issues/1321

This issue will rear it's head significantly, if still present when Angular 9 is properly released.

**Comment by hevans90** [comment]
This is the issue:

![image](https://user-images.githubusercontent.com/12142999/73170620-cd606300-40f6-11ea-83fa-df6021ce15e7.png)

Should be a trivial fix.

**Comment by swiftone** [comment]
Thanks for the detailed report @hevans90 - We'll take a look.

**Comment by swiftone** [comment]
Internal ref: OKTA-274511

**Comment by mraible** [comment]
I just created a new app with Angular CLI 9.0.0-rc.11 (released today), installed the Okta Angular SDK 1.3.1 and was able to run `ng test` and `ng e2e` without issues.

I used @oktadev/schematics to create the app and test it. You can see the passing CI build at https://travis-ci.org/oktadeveloper/schematics/builds/643544016#L540.

```
$ ng version

Angular CLI: 9.0.0-rc.11
Node: 12.13.0
OS: darwin x64

Angular: 9.0.0-rc.11
... animations, cli, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router
Ivy Workspace: Yes

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.900.0-rc.11
@angular-devkit/build-angular     0.900.0-rc.11
@angular-devkit/build-optimizer   0.900.0-rc.11
@angular-devkit/build-webpack     0.900.0-rc.11
@angular-devkit/core              9.0.0-rc.11
@angular-devkit/schematics        9.0.0-rc.11
@angular/cdk                      8.2.3
@ngtools/webpack                  9.0.0-rc.11
@schematics/angular               9.0.0-rc.11
@schematics/update                0.900.0-rc.11
rxjs                              6.5.4
typescript                        3.7.5
webpack                           4.41.2
```

**Comment by hevans90** [comment]
I can't reproduce this from a fresh install of rc-11. I should have done before opening this issue. This works fine, must have been some weirdness in the ivy compiler when moving between `rc.10` and `rc.11`.

Sorry guys!
